src/Directory.Build.props

```diff
@@ -1,5 +1,7 @@
 <Project>
   <PropertyGroup>
     <LangVersion>latest</LangVersion>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
```

Comment:
Why not in GraphQL.csproj ?

Reply:
Also it seems that `DebugSymbols` is `true` by default.

Reply:
i think it is only true in debug? and i will move this into GraphQL.csproj

Reply:
You are right, only in debug. But this page https://github.com/dotnet/sourcelink says nothing about `DebugSymbols`. It is necessary?

Reply:
https://github.com/dotnet/sourcelink#using-source-link-in-net-projects

> If you distribute the library via a package published to NuGet.org, it is recommended to build a symbol package and publish it to NuGet.org as well. This will make the symbols available on NuGet.org symbol server, where the debugger can download it from when needed. Alternatively, you can include the symbols in the main package. However, doing so is not recommended as it increases the size of the package and thus restore time for projects that consume your package.

Although i think the " However, doing so is not recommended as it increases the size of the package and thus restore time for projects that consume your package." is FUD

Using embedded symbols means the package goes from 171KB => 242KB. So yes it is bigger, but it means we always get line numbers for exceptions, even in production environments. Using a snupkg (symbols package) keeps the nuget smaller, but in my experience no one sets those up correctly to work in production environments. Many dont even have them working in dev environs, since many orgs white list packages. So i go with embedded for all my nugets, it is just simpler

Reply:
My question is not about the `DebugType` property, but about `DebugSymbols`. I understand what `DebugType` is for. I experimented a bit with building packages with different settings. I found this https://github.com/Microsoft/msbuild/issues/2169 and I see that you also left a comment. It seems that `DebugType` takes precedence over `DebugSymbols`.

I understand that all this may not be important, I just don’t want to litter the project files with insignificant settings that will cause unnecessary questions.

Reply:
yep fair enough. will remove DebugSymbols